### PR TITLE
feat(auth): add sso auth modal in container

### PR DESCRIPTION
**Pull request by zakariasahmane**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/tracking-plugin
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| License          | BSD 3-Clause

Add SSO Auth Modal in Container

### CODE REVIEW

File: packages/manager/apps/container/src/public/translations/sso-auth-modal/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "sso_modal_title": "OVH - Authentification",
```
Comment (antleblanc) [inline packages/manager/apps/container/src/public/translations/sso-auth-modal/Messages_fr_FR.json:2]:
```suggestion
  "sso_modal_title": "OVHcloud - Authentification",
```

File: packages/manager/apps/container/src/public/translations/sso-auth-modal/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "sso_modal_loading": "Chargement...",
```
Comment (antleblanc) [inline packages/manager/apps/container/src/public/translations/sso-auth-modal/Messages_fr_FR.json:3]:
```suggestion
  "sso_modal_loading": "Chargement…",
```

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.jsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+                </div>
+              </div>
+            )}
+            {(mode === disconnectedToConnected ||
+              mode === connectedToOther) && (
+              <div
+                className={
+                  mode === disconnectedToConnected
+                    ? 'col-md-12 span12'
+                    : 'col-md-6 span6'
```
Comment (antleblanc) [inline packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.jsx:106]: IMHO you can safely remove those helpers (they came from Bootstrap 2)

```suggestion
                    ? 'col-md-12'
                    : 'col-md-6'
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/components/ovh-shell/src/plugin/ux/components/modal.ts [hunk 4]
```diff
@@ -80,6 +80,22 @@ export class UxModal {
...
   }
 
+  refreshModal(options: IModalOptions) {
+    this.options.content = options.content;
+    this.elements.appendTo = document.querySelector(this.options.appendTo);
+    if (this.options.content) {
+      this.elements.body.removeChild(this.elements.body.firstElementChild);
+      this.elements.body.append(this.options.content);
+    }
+    // this.render();
```
[the remark this answers is not in the document]
Comment (Jisay) [reply packages/components/ovh-shell/src/plugin/ux/components/modal.ts]: done

File: packages/manager/apps/container/src/public/translations/sso-auth-modal/Messages_fr_FR.json (hunk 1 above)
Comment (Jisay) [reply packages/manager/apps/container/src/public/translations/sso-auth-modal/Messages_fr_FR.json]: done

File: packages/manager/apps/container/src/public/translations/sso-auth-modal/Messages_fr_FR.json (hunk 2 above)
Comment (Jisay) [reply packages/manager/apps/container/src/public/translations/sso-auth-modal/Messages_fr_FR.json]: done

File: packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.jsx (hunk 3 above)
Comment (Jisay) [reply packages/manager/apps/container/src/sso-auth-modal/SSOAuthModal.jsx]: removed: done :-)

**Review by antleblanc — CHANGES_REQUESTED** [review]
> Some checks were not successful

Could you please have a look?
Additional question: Is there Pull Request linked to an internal ticket reference?

File: packages/manager/apps/container/package.json [hunk 5]
```diff
@@ -29,13 +29,14 @@
...
     "@ovh-ux/ui-kit": "^5.1.0",
-    "bootstrap": "^4.4.1",
+    "bootstrap": "4.6.0",
     "date-fns": "^2.24.0",
     "i18next": "^20.4.0",
     "i18next-http-backend": "^1.3.0",
     "lodash-es": "^4.17.15",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.1.2",
```
Comment (Jisay) [inline packages/manager/apps/container/package.json:39]: `react-bootstrap` v2.x seems to be for `bootstrap` v5.x but you are using `bootstrap` v4.6 (https://github.com/react-bootstrap/react-bootstrap#bootstrap-compatibility)

**Review by Jisay — APPROVED** [review]

**Pull request merged**